### ISSUE TITLE
The option `browsers: ['not ie < 8']` not works

### ISSUE THREAD

**Issue by aleen42** [postcss/autoprefixer#841]
I have set up `postcss` and `autoprefix` to want to build the following CSS code:

```css
.example {
    display: flex;
    transition: all .5s;
    user-select: none;
    background: linear-gradient(to bottom, white, black);
}
```

With the following option:

```js
gulp.src('./test.css')
        .pipe(postcss([autoprefixer({ browsers: ['not ie < 8'] })]))
        .pipe(gulp.dest('./dist'));
```

But it seems not work, and why?

I have also tested with `last 4 versions`, it works! And also why? I am using the newest version of **autoprefixer@7.1.0**

**Comment by ai** [comment]
`not X` removes browsers from previous selection. But you didn't set previous selection 😊. Use `defaults, not IE < 8`.

But it is UI issue. I will add warning to Browserslist (BTW, don't set browsers to option, use `browserslist` config).

**Comment by aleen42** [comment]
Thanks for your reply, and it has been solved. 👍

**Comment by ai** [comment]
I fixed UX https://github.com/ai/browserslist/commit/bc78b5d635344796c77a7e23b66f311ad0d9ab5e

I will release it soon

**Comment by ai** [comment]
Released in 2.1.3
